lib/publiccloud/utils.pm

```diff
@@ -49,4 +49,19 @@ sub is_ondemand() {
     return is_publiccloud && (!is_byos());    # When introducing new flavors, add checks here accordingly.
 }
 
+# Check if we are on an Azure test run
+sub is_azure() {
```

Comment:
this is dead code , I agree that most probably we will need this in the future but let's commit it with first use case